src/formElementPlugins/types.ts

```diff
@@ -56,7 +56,7 @@ type ValidationState = {
 interface ApplicationViewProps extends ApplicationViewWrapperProps {
```

Comment:
I'm confused why are we still using extends `ApplicationViewWrapperProps` instead of ApplicationViewWrapperPropsNEW here? I was going to point out to the `parameters` prop not defining all these possible arguments received. I guess we might want to 1 - change to use `ApplicationViewWrapperPropsNEW` and 2 - list all these new types in a new type for `parameters` since now this is well defined.

I have started here:

```
interface QuestionParameters {
    label: string
    description?: string
    placeholder?: string
    search?: boolean
    options: string[]
    optionsDisplayProperty?: string
    default: string | number
}
```

Reply:
Yeah, I'll need to update this once you've merged #568 I think. I'll try and review that this morn so we can merge it then I'll pull it into here and tidy up.

Reply:
Okay, I've merged from the new master now, so this should be fine I think.